tests/ClientFunctionalTest.php

```diff
@@ -277,7 +279,7 @@ public function testDeleteIdsIdxString(): void
         // Refresh index
         $index->refresh();
 
-        $resultData = $result->getData();
```

Comment:
If we would have a wrapper around result, get we keep getData as a proxy for asArray?